Signal/src/ViewControllers/ConversationView/Cells/OWSMessageBubbleView.m

```diff
@@ -230,6 +230,7 @@ - (void)configureViews
     OWSAssert([self.viewItem.interaction isKindOfClass:[TSMessage class]]);
     OWSAssert(self.contentWidth > 0);
 
+    CGSize quotedMessageContentSize = [self quotedMessageSizeForContentWidth:self.contentWidth includeMargins:NO];
```

Comment:
Fix a bug. We now ensure that "quoted message," "media view," and "body text view" all correctly share space with each other.